src/Microsoft.OpenApi/Validations/ValidationRuleSet.cs

```diff
@@ -110,11 +110,11 @@ public ValidationRuleSet(IEnumerable<ValidationRule> rules)
         /// <summary>
         /// Gets the rules in this rule set.
         /// </summary>
-        public IList<ValidationRule> Rules
+        public IEnumerable<ValidationRule> Rules
```

Comment:
_rules and empty rules could both be readonly, would you mind correcting this as well while at it?